docs/components/file-tree/index.tsx

```diff
@@ -112,14 +114,14 @@ const File: React.FC<FileProps> = React.memo(
   ({ label, name, active, ...props }) => {
     return (
       <li
-        className={cn('flex list-none', {
-          ['nx-text-primary-600 contrast-more:!nx-text-primary-600 contrast-more:nx-text-gray-900 contrast-more:nx-underline contrast-more:dark:nx-text-gray-50']:
-            active
-        })}
+        className={cn(
+          'flex list-none',
+          active && 'nx-text-primary-600 contrast-more:nx-underline'
+        )}
```

Comment:
`contrast-more:nx-text-gray-900` and `contrast-more:!nx-text-primary-600` contradicts, they can be safely removed as and `contrast-more:dark:nx-text-gray-50`, I tried on Vercel with `contast-more` mode and it looks the same as in nextra.site